js-miniapp-sdk/src/types/responseTypes/AdResponse.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * display date is ad's load time
```

Comment:
Is this the time that the Ad was _loaded_ or _displayed_? Ads will be loaded when the Mini App is launched and then displayed immediately after the Mini App calls the `showInterstitial` function. 

Either way, I'm still not sure I understand what the use case is for this? It seems like this is a value that the Mini App would be able to find out on their own.

Reply:
Removed this prop for now, if in the future we get any requirements regarding display time, we may consider then!
Thanks!